tests/rbd/test_rbd_trash_purge_schedule.py

```diff
@@ -0,0 +1,159 @@
+import random
+import time
+
+from ceph.parallel import parallel
+from tests.rbd.rbd_utils import Rbd, initial_rbd_config
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+
+def create_an_image_run_fio(rbd, pool, client, image_name):
+    if rbd.create_image(pool, image_name, "10G"):
+        log.error(f"Image creation failed for {image_name}")
+        return 1
+    run_fio(image_name=image_name, pool_name=pool, client_node=client, runtime=10)
+    return 0
+
+
+def create_images_run_fio_in_parallel(rbd, pool, client, num_images):
+    log.info(f"Creating {num_images} images in pool {pool} and initiating io")
+
+    with parallel() as p:
+        for i in range(0, num_images):
+            p.spawn(create_an_image_run_fio, rbd, pool, client, f"image_{i}")
+        return 0
+
+
+def move_all_images_to_trash(rbd, pool, num_images):
+    log.info("Moving all images in pool to trash")
+    for i in range(0, num_images):
+        rbd.move_image_trash(pool, f"image_{i}")
+
+
+def prepare_images_and_move_to_trash(rbd, pool, client, num_images):
+    create_images_run_fio_in_parallel(rbd, pool, client, num_images)
+    move_all_images_to_trash(rbd, pool, num_images)
+
+
+def verify_schedule_add(
+    rbd: Rbd,
+    verification_iterations: int,
+    schedule_interval: int,
+    pool: str,
+    client,
+    num_images: int,
+) -> int:
+    iterations_remaining = verification_iterations
+    log.debug(f"Verifying purge on pool {pool}")
+    while iterations_remaining > 0:
+        log.debug(f"Sleeping for {schedule_interval} mins with  buffer")
```

Comment:
I think schedule_interval is not in min as per the method in rbd_util trash_purge_schedule(), its in sec.
correct me if i am wrong. or is it 4 min ?

Reply:
Yes, Even I got confused
But later checked that trash purge schedule is accepting argument in minutes (unlike some other commands expecting in seconds)

Planning to raise an RFE asking for clear doc in rbd help command

Reply:
Changed.